lib_free_u/unet.py

```diff
@@ -99,10 +100,12 @@ def forward(self, x, timesteps=None, context=None, y=None, **kwargs):
 
 def free_u_cat(h, h_skip):
     if h.shape[1] == 1280:
-        h[:, :640] = h[:, :640] * global_state.backbone_factors[0]
+        offset = floor(640*global_state.backbone_offset[0])
```

Comment:
You think making this 1280 and make it wrap around so that offset=0 is the same thing as offset=1 could be a good idea?

Reply:
I'm not python dev, so not sure. If it works like that feel free to change it. I had to chatgpt floor method in python xD

Reply:
ah okay! no problem. just a note when you make a PR in a repo in the future, the name of your branch is `main` so I have to overwrite my local reference to the `main` branch to work on this lol.